server/locale/DE/Pi30/mutations/pi30.js

```diff
@@ -0,0 +1,142 @@
+import Logo from '../../../../message/logos';
+import { xSmallFallback, textWrap, messageLogoWidth, altNoWrap, setLogoTop } from '../../../../message/mediaQueries';
+import { xSmallNoWrap } from './mediaQueries';
+import { textLogoMutations } from '../../../../message/logoMutations';
+
+export default {
+    'layout:text': [
+        [
+            'default',
+            ({ textSize }) => ({
+                styles: [
+                    textWrap(textSize * 55, textSize, 'DE'),
+                    xSmallFallback(textSize * 15),
+                    messageLogoWidth(false, textSize * 4, textSize * 1.25),
+                    setLogoTop(textSize * 20),
+                    xSmallNoWrap(textSize * 14.2),
+                    `.message__headline > .tag--medium > span > span:last-child::after {content: '.'}`,
+                    `@media screen and 
+                    (min-width: ${textSize * 23.8}px),
+                    (max-width: ${textSize * 21.8}px) {
+                        .message__headline > .tag--medium > span > span.br:last-child {white-space: normal;}
+                    }`
+                ],
+                logo: Logo.PP_PAYPAL.COLOR,
+                headline: [
+                    {
+                        tag: 'medium',
+                        br: ['Sie']
```

Comment:
```suggestion
                        br: ['bei']
```
Let's change all instances in this file where the br is `br: ['Sie']` and `br: ['Sie', 'Raten']` in tag medium to the above so the message will break on the correct word.